Examples/test-suite/member_pointer.i

```diff
@@ -5,6 +5,9 @@
 #pragma error_messages (off, badargtype2w) /* Formal argument ... is being passed extern "C" ... */
 #pragma error_messages (off, wbadinit) /* Using extern "C" ... to initialize ... */
 #pragma error_messages (off, wbadasg) /* Assigning extern "C" ... */
+#elif __GNUC__ >= 5 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 8)
+/* passing NULL into SWIG constant member function wrapper in Fortran raises a warning because it's an int rather than a nullptr_t */
+#pragma GCC diagnostic ignored "-Wconversion-null"
 #endif
```

Comment:
Looks this warning suppression just added is because the brute_cast doesn't work and I'm not convinced it is a good idea to suppress this. Looking at brute_cast, it is quite contradictory to the SWIG code generation philosophy outlined at [SWIG and freedom](http://swig.org/Doc3.0/Introduction.html#Introduction_nn13).
Is there any reason why the tried and tested SWIG_PackData and SWIG_UnpackData used by the other languages is not suitable? Probably with additional complications to brute_cast, null pointers can be made to work, but I'd rather Fortran code generation switched to what we normally use for maximum portability, especially C++11 should not be used unless the code being parsed is C++11.

Reply:
I see that now. When I started working on this fork I didn't understand that part of the SWIG philosophy, and I wasn't aware of the SWIG_PackData when I initially wrote the memfunptr code. I'm looking at SWIG_PackData now; wouldn't it be simpler still to just copy the raw data into a character buffer rather than going through the trouble to encode and decode it into an ASCII hex string? Whatever the case, I'll remove the `brute_cast` code.

Reply:
Warning is no longer necessary since I robustified how constant data such as member function pointers are wrapped by default.